config/settings/base.py

```diff
@@ -375,4 +376,8 @@
 OSLER_ATTENDANCE_URL = env(
     "OSLER_ATTENDANCE_URL",
     default="https://www.wustl.edu",
-)
\ No newline at end of file
+)
+
+#Default Encounter Status 
+OSLER_DEFAULT_ACTIVE_STATUS = ('Active', True)
+OSLER_DEFAULT_INACTIVE_STATUS = ('Inactive', False)
```

Comment:
Previously saved encounters will be fine? Things are filtered by the boolean is_active or not not by the name. These are settings so you can change the name. When changed, previous encounters will still have the name of the old status but new encounters going forward will toggle between the new ones you pick

Like in the future if you wanted your default active status of when you activate a patient to be Arrived in Clinic, this would let you change that. And you could have multiple EncounterStatus that isn't just a boolean toggle

Reply:
Got it now.